### DISCUSSION TITLE
Deployment of packaged Qt app (without Visual Studio project)

### DISCUSSION THREAD

**Discussion by triplef** [microsoft/WindowsAppSDK#2139] [Q&A] [answered]
We managed to integrate the Windows App SDK into our Qt app thanks to the help in https://github.com/microsoft/WindowsAppSDK/discussions/1891, and it works fine when we run the app unpackaged and call `MddBootstrapInitialize()` at launch.

However, we want to distribute as a **packaged** app in the Microsoft Store, but we’re running into some issues with the deployment following [this documentation](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/deploy-packaged-apps). Here is what we did so far:

- We added a package dependency to `Microsoft.WindowsAppRuntime.1.0` in our `AppxManifest.xml` (not sure if this is correct?):
  ```xml
  <Dependencies>
    <TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.18362.0" MaxVersionTested="10.0.19044.0" />
    <PackageDependency Name="Microsoft.VCLibs.140.00.UWPDesktop" MinVersion="14.0.24217.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" />
    <PackageDependency Name="Microsoft.WindowsAppRuntime.1.0" MinVersion="0.318.928.0" Publisher="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" />
  </Dependencies>
  ```
- At runtime, we’re calling `DeploymentManager::GetStatus()`, which returns `DeploymentStatus::PackageInstallRequired`.
- So we call `DeploymentManager::Initialize()`, which returns `DeploymentStatus::PackageInstallFailed` with extended error 0x800700E8 (ERROR_NO_DATA).

I’d appreciate any help to get this working.

**Comment by DrusTheAxe** [comment]
@sachintaMSFT is the `ERROR_NO_DATA` failure another symptom of the inconsistent-manifest problem you're working on?

**Reply by sachintaMSFT** [reply]
ERROR_NO_DATA is not the manifest mis-match problem (that's different error).

**Comment by triplef** [comment, marked as answer]
This is now working correctly with the latest v1.1 of the Windows App SDK.
